templates/CRM/Contribute/Form/Search/ContributionRecur.tpl

```diff
@@ -35,40 +35,28 @@
         <td colspan="4">{$form.contribution_recur_payment_made.html}</td>
       </tr>
       <tr>
-        <td>{ts}Start Date{/ts}</td>
```

Comment:
It would be nice to iterate through an array for these - just to make it clear they are all boilerplate  & discourage random tweaking - it would be a step towards the whole tpl being generic which looks very possible for this one